images/bootstrap/runner

```diff
@@ -16,7 +16,21 @@
 set -o errexit
 set -o nounset
 set -o pipefail
+set -x
 
+# Check if the job has opted in to docker-in-docker availability.
+if [[ -z "$DOCKER_IN_DOCKER_ENABLED" ]]; then
+# If not, make sure `docker` points to the old one compatible with our Jenkins
+    export PATH=/docker-no-dind-bin/:$PATH
+else
+# If we have opted in to docker in docker, start the docker daemon.
+# sometimes there may be errors about cgroups already being mounted,
+# these should not be a problem.
+    echo "Please ignore benign 'cgroup is already mounted' errors here :-)"
```

Comment:
`benign`?

Reply:
Yes they look scary but aren't actually a problem, when it happens it happens during starting the docker service.